kolibri/plugins/coach/assets/src/app.js

```diff
@@ -10,59 +10,55 @@ const store = require('./state/store');
 const PageNames = require('./state/constants').PageNames;
 
 
-const REPORT_URL_PATTERN = [
-  '/reports',
-  ':channel_id',
-  ':content_scope',
-  ':content_scope_id',
-  ':user_scope',
-  ':user_scope_id',
-  ':all_or_recent',
-  ':view_by_content_or_learners',
-  ':sort_column',
-  ':sort_order',
-].join('/');
-
-
 class CoachToolsModule extends KolibriModule {
   ready() {
     coreActions.getCurrentSession(store).then(() => {
       const routes = [
         {
-          name: PageNames.REPORTS_NO_QUERY,
-          path: '/reports',
+          name: PageNames.COACH_CLASS_LIST_PAGE,
+          path: '/classes',
           handler: (toRoute, fromRoute) => {
-            actions.redirectToDefaultReport(store, toRoute.params);
+            actions.showClassListPage(store);
           },
         },
         {
-          name: PageNames.REPORTS_CHANNEL,
-          path: '/reports/:channel_id',
+          name: PageNames.COACH_RECENT_PAGE,
+          path: '/classes/:id/recent',
```

Comment:
I think `classes` (plural) is misleading, since it's indicating a particular class